example/make/unix_like.mk

```diff
@@ -43,18 +43,14 @@ build: $(OBJECTS)
 	@echo
 	@echo Build successful!
 
-%: $(DIR_SRC)/%.cpp 
+%: $(DIR_SRC)/%.cpp
 ifeq ($(ENUMARATE), true)
 	$(eval COUNT_OBJ=$(shell echo $$(($(COUNT_OBJ)+1))))
 	@echo [$(COUNT_OBJ)/$(COUNT_SRC)] Compiling: $<
 else
 	@echo Compiling: $<
 endif
 	@$(CXX) $< -o $(DIR_EXE)/$@ $(CXX_ARGS)
-ifeq ($(ENUMARATE), false)
-	@echo Build successful!
```

Comment:
I liked the info about build successful but if you think it's not necessary then okay.